packages/shared/src/components/cards/survey/FeedSurveyCard.tsx

```diff
@@ -21,21 +21,21 @@ export function FeedSurveyCard({
 
   return (
     <div
-      className="relative rounded-16 p-px"
+      className="relative rounded-16 p-0.5"
       style={{
         backgroundImage:
           'linear-gradient(180deg, rgba(255, 233, 35, 1) 0%, rgba(252, 83, 141, 1) 50%, rgba(113, 71, 237, 1) 100%)',
       }}
     >
-      <div className="absolute inset-px rounded-16 bg-background-default" />
       <CardComponent
         className={classNames(
-          'relative items-center gap-4 overflow-hidden',
+          '!bg-background-default/[.92] shadow-none relative items-center gap-4 overflow-hidden !rounded-14 border-0',
```

Comment:
`!bg-background-default/[.92] shadow-none` should've worked according to Tailwind docs, but they didn't locally. So I added their equivalent in `style`

Reply:
Do we still need to keep this here since it doesn't work?

Reply:
I removed it. And fixed so `shadow-none` works